plugins/preview/plugin.js

```diff
@@ -155,6 +158,14 @@
 
 			return html;
 		}
+
+		function setPrintCallback( callback ) {
+			if ( !callback ) {
+				return '';
+			}
+
+			return '<script> window.onreadystatechange = firePrint </script>';
```

Comment:
It should be bound to `document`, because `readystatechange` event does not bubble up to `window`.

```suggestion
			return '<script> document.onreadystatechange = firePrint; </script>';
```